src/scheduler/prime.c

```diff
@@ -467,16 +501,26 @@ load_day(enum days d, enum prime_time pr, char *tok)
 
 	if (tok != NULL) {
 		if (!strcmp(tok, "all") || !strcmp(tok, "ALL")) {
+			if (pr == NON_PRIME && conf.prime[d][PRIME].all == TRUE) {
+				schdlog(PBSEVENT_SCHED, PBS_EVENTCLASS_FILE, LOG_NOTICE, HOLIDAYS_FILE,
+						"Warning: both prime & non-prime starts are 'all'; assuming 24hr primetime");
+				return 0;
+			}
 			conf.prime[d][pr].all = TRUE;
 			conf.prime[d][pr].hour = UNSPECIFIED;
 			conf.prime[d][pr].min = UNSPECIFIED;
-		}
-		else if (!strcmp(tok, "none") || !strcmp(tok, "NONE")) {
-			conf.prime[d][pr].none = TRUE;
+			conf.prime[d][pr].none = FALSE;
+		} else if (!strcmp(tok, "none") || !strcmp(tok, "NONE")) {
```

Comment:
I'm not sure if values like NoNe or NOne are supposed to be allowed. Since you left it up to me, I'd like to keep the code as is for now. Thanks for being diligent :)

Reply:
+1 for strcasecmp()
+2 for strncasecmp()